### PR TITLE
Add config to disable block cache

### PR DESCRIPTION
**Pull request by huachaohuang**
This is mainly for test, so I think we don't need to add it to the template.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/config.rs [hunk 1]
```diff
@@ -92,6 +93,7 @@ macro_rules! build_cf_opt {
     ($opt:ident) => {{
         let mut block_base_opts = BlockBasedOptions::new();
         block_base_opts.set_block_size($opt.block_size.0 as usize);
+        block_base_opts.set_no_block_cache($opt.disable_block_cache);
```
Comment (zhangjinpeng1987) [inline src/config.rs]: What will happen if `no_block_cache = true` and `block_size` is not 0?

Comment (huachaohuang) [reply src/config.rs]: RocksDB will reset the block cache if `no_block_cache = true`.

**Pull request merged**